specifyweb/frontend/js_src/lib/components/Merging/index.tsx

```diff
@@ -361,6 +393,99 @@ function MergeButton<SCHEMA extends AnySchema>({
   );
 }
 
+type StatusState = {
+  status: string;
```

Comment:
This status can only be one of three strings, correct? 
Can you narrow its type then to:
```suggestion
  status: 'MERGING' | 'SUCCESS' | 'FAILED';
```